kloppy/domain/models/common.py

```diff
@@ -6,23 +6,34 @@
 from .pitch import PitchDimensions
 
 
-class Team(Enum):
-    HOME = "home"
-    AWAY = "away"
-
-    def __str__(self):
-        return self.value
+@dataclass
+class Position:
+    position_id: str
+    name: str
+    coordinates: List
 
 
 @dataclass
 class Player:
     player_id: str
-    team: Team
     name: str
+    first_name: str
+    last_name: str
     jersey_no: str
+    position: Position
```

Comment:
not sure what the issue is here. What are you referring to by *using the same name for different things*? By using position for the Player attribute and Position for the model? If so I think it makes sense and it's actually clearer that the position attribute is a Position class. Otherwise might lead to confusion.

Other than here, I don't think position is used anywhere else, or is it?

Reply:
see below